_edgedb/loaddata.py

```diff
@@ -36,8 +36,7 @@ def _start(self):
                 asyncio.create_task(self._worker()))
 
     async def _worker(self):
-        con = await edgedb.async_connect(
-            user='edgedb', database='edgedb_bench')
+        con = await edgedb.async_connect('edgedb_bench')
```

Comment:
Drop explicit instance name, since we use `edgedb project` now.

Reply:
I'm inclined to keep it for sanity and consistency reasons for now. There are a few places where the instance name is used explicitly:
- Some benchmarks that don't reside in the `_edgedb` project directory (`_edgedb_js` and `_go`). This probably can be addressed by either reshuffling them or placing `edgedb.toml` to the root instead of the `_edgedb` sub-directory. Mostly each benchmark was setup in its own directory as if it were a separate project, so that's why the arrangement is like it is at the moment.
- The HTTP benchmarks need the port where EdgeDB is exposing GraphQL and EdgeQL in order to run. They don't use any fancy drivers instead just comparing raw HTTP access. Currently, I use `edgedb server status --json edgedb_bench` to get the port where EdgeDB is exposing the needed endpoints. I don't think we have a programmatic way of getting this. Also, running a query like `select cfg::get_config_json();` only shows a 5656 port, as I assume that's how it's setup _inside_ docker, which is not terribly helpful in figuring out the actual port that the instance exposes.

Reply:
The trouble with hardcoded instance name is that you must remember to set it up properly when `edgedb project init` is ran.  I made a mistake of accepting the default ("edgedb") and things broke in various ways.

> The HTTP benchmarks need the port where EdgeDB is exposing GraphQL and EdgeQL in order to run.

`edgedb project info` can be used to obtain the name of the instance associated with the project.

Reply:
For the time being I actually updated the Makefile to include the instance name `edgedb_bench` when generating a project. I also included instructions in the readme to actually use that name. Maybe I'll make another pass at the setup and benchmark code to completely remove the dependency on the specific instance name and sniff it with `edgedb project info` so that I can use it where needed.